src/aind_slims_api/models/base.py

```diff
@@ -77,5 +76,17 @@ def _serialize(self, field, info):
         else:
             return field
 
+    def model_dump(self, serialize_quantity=True, *args, **kwargs) -> Dict[str, Any]:
```

Comment:
It might not be a good idea to override pydantic methods. Is there a different way to handle this?

Reply:
I could make it a custom method instead -- something like "custom_model_dump"?

Reply:
Okay, actually I have new thoughts on this. The gymnastics are necessary because we have a special serialization need for quantities when posting to SLIMS, implemented with the _serialize method. However, we also want to be able to serialize naturally into a python dict with reasonable values, but the departure from the norm is due to slims, so whatever we do, the normal behavior should be default and the weird behavior should be different.

All that to say, here's the three options I see:
1. Change the validator to accept a quantity as a dict
2. add a model_dump_slims method (analogous to pydantic's model_dump_json) that does the weird serialization.
3. Add [context ](https://docs.pydantic.dev/latest/concepts/serialization/#serialization-context) to the serializer so the client can call `.model_dump(context='slims_post')` and get the quantity dict, but a normal `**model` will remain the types you expect.

I like 3 the best, but we could also do 2 and 3 together to add clarity.

Reply:
3 seems reasonable. We are going to ask Slims to provide a plugin to let us just query their sql db tables directly. I feel like we are having to resort to too many of these "gymnastics" to just get data from their backend. It might almost be more efficient to just use the `requests` module directly.

Reply:
Less gymnastics seems better. I think the main goal of this library is to just isolate the ways people get and put data into SLIMS so that people are using versioned code (that can be improved and tracked with issues, etc) rather than passing around requests snippets. It would be really helpful if it was easier to use but I think the Session, Mouse interfaces are a huge improvement in the right direction.